Dockerfile

```diff
@@ -91,6 +86,8 @@ COPY --chown=node:node --from=builder /shieldbattery/package.json /shieldbattery
 COPY --chown=node:node --from=builder /shieldbattery/babel-register.js /shieldbattery/babel-register.js ./
 COPY --chown=node:node --from=builder /shieldbattery/server/deployment_files/entrypoint.sh /entrypoint.sh
 
+RUN apk --no-cache add s3cmd
```

Comment:
Think this needs to be done earlier, before we change the user to node (probably could be added to line 56?)